components/profile/fields/ProfileFields.tsx

```diff
@@ -0,0 +1,253 @@
+import { useState } from "react";
+import { Field, ErrorMessage } from "formik";
+import useSupabase from "../../../hooks/useSupabase";
```

Comment:
Don't worry...absolute imports are on the roadmap :sweat_smile:

Reply:
🔥